wordpress-mobile/WordPress-Android
# Reader move tab layout to parent part 5.1 move toolbar

Partially fixes #11849

Merge instructions:

1. Review this PR
2. ~Make sure https://github.com/wordpress-mobile/WordPress-Android/pull/11890 is merged~
3. ~Change the target branch to master-feature-branch/move-reader-tablayout-to-parent~
4. ~Remove the "Not Ready for merge" label~
5. Merge this PR

This is another PR in a series of PRs which is trying to introduce a regular TabLayout+ViewPager2 in the main Reader view. The current implementation uses FilteredRecyclerView component that has some downsides (eg. swipe gesture doesn't work). These changes will also help us implement the improvements in the ReaderImprovements project.

P.S. The known issues/remaining tasks are listed in #11849

Goal of this PR is to introduce Toolbar on the ReaderFragment. Also fixes color of the Following-subfilter component in the dark mode.

To test:
1. Open the reader and make sure the toolbar and its title is visible
2. Scroll down and notice the toolbar collapses

PR submission checklist:
